zenoh-kotlin/src/commonMain/kotlin/io/zenoh/prelude/Encoding.kt

```diff
@@ -24,81 +24,92 @@ package io.zenoh.prelude
+        val APPLICATION_PROTOBUF = Encoding(17, description = "application/protobuf")
+        val APPLICATION_JAVA_SERIALIZED_OBJECT = Encoding(18, description = "application/java-serialized-object")
+        val APPLICATION_OPENMETRICS_TEXT = Encoding(19, description = "application/openmetrics-text")
+        val IMAGE_PNG = Encoding(20, description = "image/png")
+        val IMAGE_JPEG = Encoding(21, description = "image/jpeg")
+        val IMAGE_GIF = Encoding(22, description = "image/gif")
+        val IMAGE_BMP = Encoding(23, description = "image/bmp")
+        val IMAGE_WEBP = Encoding(24, description = "image/webp")
+        val APPLICATION_XML = Encoding(25, description = "application/xml")
+        val APPLICATION_X_WWW_FORM_URLENCODED = Encoding(26, description = "application/x-www-form-urlencoded")
+        val TEXT_HTML = Encoding(27, description = "text/html")
+        val TEXT_XML = Encoding(28, description = "text/xml")
+        val TEXT_CSS = Encoding(29, description = "text/css")
+        val TEXT_JAVASCRIPT = Encoding(30, description = "text/javascript")
+        val TEXT_MARKDOWN = Encoding(31, description = "text/markdown")
+        val TEXT_CSV = Encoding(32, description = "text/csv")
+        val APPLICATION_SQL = Encoding(33, description = "application/sql")
+        val APPLICATION_COAP_PAYLOAD = Encoding(34, description = "application/coap-payload")
+        val APPLICATION_JSON_PATCH_JSON = Encoding(35, description = "application/json-patch+json")
+        val APPLICATION_JSON_SEQ = Encoding(36, description = "application/json-seq")
+        val APPLICATION_JSONPATH = Encoding(37, description = "application/jsonpath")
+        val APPLICATION_JWT = Encoding(38, description = "application/jwt")
+        val APPLICATION_MP4 = Encoding(39, description = "application/mp4")
+        val APPLICATION_SOAP_XML = Encoding(40, description = "application/soap+xml")
+        val APPLICATION_YANG = Encoding(41, description = "application/yang")
+        val AUDIO_AAC = Encoding(42, description = "audio/aac")
+        val AUDIO_FLAC = Encoding(43, description = "audio/flac")
+        val AUDIO_MP4 = Encoding(44, description = "audio/mp4")
+        val AUDIO_OGG = Encoding(45, description = "audio/ogg")
+        val AUDIO_VORBIS = Encoding(46, description = "audio/vorbis")
+        val VIDEO_H261 = Encoding(47, description = "video/h261")
+        val VIDEO_H263 = Encoding(48, description = "video/h263")
+        val VIDEO_H264 = Encoding(49, description = "video/h264")
+        val VIDEO_H265 = Encoding(50, description = "video/h265")
+        val VIDEO_H266 = Encoding(51, description = "video/h266")
+        val VIDEO_MP4 = Encoding(52, description = "video/mp4")
+        val VIDEO_OGG = Encoding(53, description = "video/ogg")
+        val VIDEO_RAW = Encoding(54, description = "video/raw")
+        val VIDEO_VP8 = Encoding(55, description = "video/vp8")
+        val VIDEO_VP9 = Encoding(56, description = "video/vp9")
```

Comment:
Any reason to stop at `56`? https://github.com/eclipse-zenoh/zenoh/blob/926208cd2bd9a7ef59e6151ff17a99ce6b78edbd/zenoh/src/api/encoding.rs#L641

Reply:
I just updated the list.